test/mirage/help/describe.t

```diff
@@ -0,0 +1,332 @@
+             unikernel has already been configured.
+  
+         --no-eval
+             Do not evaluate the graph before showing it. See --eval. The
+             default when the unikernel has not been configured.
+  
+  CONFIGURE OPTIONS
+         --context-file=FILE (absent=mirage.context)
+             The context file to use.
+  
+         --dry-run
+             Display I/O actions instead of executing them.
+  
+         -f FILE, --file=FILE, --config-file=FILE (absent=config.ml)
+             The configuration file to use.
+  
+         -o FILE, --output=FILE
+             Name of the output file.
+  
+  APPLICATION OPTIONS
+         --hello=VAL (absent=Hello World!)
+             How to say hello. 
+  
+  OPTIONS
+         --help[=FMT] (default=auto)
+             Show this help in format FMT. The value FMT must be one of `auto',
+             `pager', `groff' or `plain'. With `auto', the format is `pager` or
+             `plain' whenever the TERM env var is `dumb' or undefined.
+  
+         --version
+             Show version information.
+  
+  COMMON OPTIONS
+         --color=WHEN (absent=auto)
+             Colorize the output. WHEN must be one of `auto', `always' or
+             `never'.
+  
+         -q, --quiet
+             Be quiet. Takes over -v and --verbosity.
+  
+         -v, --verbose
+             Increase verbosity. Repeatable, but more than twice does not bring
+             more.
+  
+         --verbosity=LEVEL (absent=warning)
+             Be more or less verbose. LEVEL must be one of `quiet', `error',
+             `warning', `info' or `debug'. Takes over -v.
+  
+  ENVIRONMENT
+         These environment variables affect the execution of describe:
+  
+         MIRAGE_LOGS
+             See option --logs.
+  
+         MODE
+             See option --target.
+  
+
+Help describe --help=plain
+  $ ./config.exe describe --help=plain | tee d2
+  NAME
+         mirage-describe - Describe a mirage application.
+  
+  SYNOPSIS
+         mirage describe [OPTION]... 
+  
+  DESCRIPTION
+         The describe command describes the configuration of a mirage
+         application.
+  
+         The dot output contains the following elements:
+         If vertices
+             Represented as circles. Branches are dotted, and the default
+             branch is in bold.       Configurables
+                                          Represented as rectangles. The order
+                                          of the output arrows is the order of
+                                          the functor arguments.       
+                                                                Data
+                                                                dependencies
+                                                                           
+                                                                Represented as
+                                                                dashed arrows.
```

Comment:
I think that such layout is not expected and it's probably a bug. We should make an issue about that because I don't think that your PR introduce this bug.

Reply:
Yes, looking at the diff the layout issue was already present in the `test/mirage/help/describe.expected` file.